services/explorer/src/api.rs

```diff
@@ -545,32 +544,32 @@ impl ExplorerApi {
     }
 
     /// Returns call status of `before_transactions` hook.
-    fn before_transactions_status(
-        schema: Schema<&dyn Snapshot>,
+    async fn before_transactions_status(
+        state: ServiceApiState,
```

Comment:
Just to be clear: Did you change the signature because `async fn` automatically includes argument lifetimes into the output `Future`? You can still use an ordinary `fn` with an `impl Future` return value 🙃

Reply:
Imo in this case it doesn't make sense, because this handlers anyway take `ServiceApiState` by value. Moreover api state cloning is cheap operation: it just increments two atomic counters and clones two strings.